src/main/webapp/app/exercises/shared/exercise-hint/manage/exercise-hint-update.component.html

```diff
@@ -32,6 +32,15 @@ <h2 id="jhi-exercise-hint-heading" [jhiTranslate]="'artemisApp.exerciseHint.home
                         <option *ngFor="let task of tasks" [ngValue]="task">{{ task.taskName }}</option>
                     </select>
                 </div>
+
+                <div class="form-group">
+                    <div>
+                        <label class="label-narrow" jhiTranslate="artemisApp.exerciseHint.displayThreshold" for="field_description">Display Threshold</label>
+                        <jhi-help-icon placement="auto" text="artemisApp.exerciseHint.displayThresholdTooltip"></jhi-help-icon>
+                    </div>
+                    <input type="number" required min="0" class="form-control" name="displayThreshold" id="field_displayThreshold" [(ngModel)]="exerciseHint.displayThreshold" />
```

Comment:
The threshold value is restricted to max. 100 on the server-side. Would it make sense to also force it here?